README.md

```diff
@@ -51,29 +52,28 @@ ONNX Runtime supports both CPU and GPU. Using various graph optimizations and ac
 
 Currently ONNX Runtime supports the following accelerators:
 * MLAS (Microsoft Linear Algebra Subprograms)
-* [MKL-DNN](./docs/execution_providers/MKL-DNN-ExecutionProvider.md) - [subgraph optimization](./docs/execution_providers/MKL-DNN-Subgraphs.md)
-* MKL-ML
+* NVIDIA CUDA
+* Intel MKL-ML
+* [Intel MKL-DNN](./docs/execution_providers/MKL-DNN-ExecutionProvider.md) - [subgraph optimization](./docs/execution_providers/MKL-DNN-Subgraphs.md)
 * [Intel nGraph](./docs/execution_providers/nGraph-ExecutionProvider.md)
-* CUDA
-* [TensorRT](./docs/execution_providers/TensorRT-ExecutionProvider.md)
-* [OpenVINO](./docs/execution_providers/OpenVINO-ExecutionProvider.md)
-* [Nuphar](./docs/execution_providers/Nuphar-ExecutionProvider.md)
+* [NVIDIA TensorRT](./docs/execution_providers/TensorRT-ExecutionProvider.md)
+* [Intel OpenVINO](./docs/execution_providers/OpenVINO-ExecutionProvider.md)
+* [Nuphar Model Compiler](./docs/execution_providers/Nuphar-ExecutionProvider.md)
 
-Not all variations are supported in the [official release builds](#apis-and-official-builds), but can be built from source following [these instructions](./BUILD.md). Find Dockerfiles [here](./dockerfiles).
+Not all variations are supported in the [official release builds](#apis-and-official-builds), but can be built from source following [these instructions](./BUILD.md).
 
 We are continuously working to integrate new execution providers for further improvements in latency and efficiency. If you are interested in contributing a new execution provider, please see [this page](docs/AddingExecutionProvider.md).
 
 
 ## Cross Platform
-[API documentation and package installation](#installation)
+ONNX Runtime is currently available for Linux, Windows, and Mac with Python, C#, C++, and C APIs. Please see [API documentation and package installation](#installation). 
```

Comment:
May consider adding Android here.

Reply:
we can add it once we've confirmed NN API readiness.